### PR TITLE
Jade Support

### PR DESCRIPTION
**Pull request by m3kka**
Added jade support to the plugin, it may be useful for someone else also

### CODE REVIEW

**Comment by ludohenin** [comment]
@m3kka thanks a lot for the PR! Could you add the test for Jade ?

**Comment by m3kka** [comment]
@ludohenin Sorry completely forgot about tests.
Decided to split the files as this is a completely new template type, let me know if you agree or not, can update it again :)

File: test/fixtures/templates-jade.js [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+@View({ templateUrl: './app.jade', styleUrls: ['./app.css', './common.css'], directives: [CORE_DIRECTIVES] })
+export class App {}
+
+// TEST_8
+@Component({
+  selector: 'app'
+})
+@View({ templateUrl: './app.jade',
+  styleUrls: ['./app.css', './common.css'], directives: [CORE_DIRECTIVES] })
+export class App {}
```
Comment (ludohenin) [inline test/fixtures/templates-jade.js]: Why did you remove tests 9 & 10 ?

**Comment by ludohenin** [comment]
Looks fine with separated files.

File: test/fixtures/templates-jade.js (hunk 1 above)
Comment (m3kka) [inline test/fixtures/templates-jade.js]: Thought they were not needed for the jade part as they run identically on the html part (I guessed they were sanity checks for components that need no including process)

**Comment by ludohenin** [comment]
It's published to npm (v0.0.8).

**Comment by m3kka** [comment]
Thank you :)
